newshomepages/save_html.py

```diff
@@ -0,0 +1,130 @@
+import typing
+from pathlib import Path
+
+import click
+from playwright.sync_api import sync_playwright
+from retry import retry
+from rich import print
+
+from . import utils
+
+
+@click.command()
+@click.argument("handle")
+@click.option("-o", "--output-dir", "output_dir", default="./")
+@click.option("-w", "--wait", "wait", default=5000)
+@click.option("-x", "--width", "width", default=1300)
+@click.option("-y", "--height", "height", default=1600)
+@click.option(
+    "-f",
+    "--full-page",
+    "full_page",
+    is_flag=True,
+    default=False,
+    help="Lazy load all images, just to make sure they're in the final single-page html",
+)
```

Comment:
Does this command need all these options?

Reply:
probably doesn't need `-w`, `-x,` `-y`. however, the `full-page` is important for lazy-loading the images. otherwise they're not included in the full-page html

Reply:
Then I would just make it always `True` and cut the option.